pages/index.md

```diff
@@ -3,16 +3,37 @@ title: NRAU Baltic
 titleTemplate: false
 ---
 
-# NRAU Baltic contest
+## NRAU Baltic contest
```

Comment:
I would suggest keeping h1 here. At least one h1 per page is recommended for SEO and accessibility

Reply:
H1 is missing on the top level of the page (next to the logo) 

It makes navigation weird because once you navigate outside main page, it doesn't mention NRAU anywhere.